lib/active_model/serializer.rb

```diff
@@ -338,7 +338,7 @@ def to_json(*args)
     # object including the root.
     def as_json(options={})
       options ||= {}
-      if root = options.fetch(:root, @options.fetch(:root, root_name))
+      if root = options.to_hash.fetch(:root, @options.fetch(:root, root_name))
```

Comment:
@mhuggins s/my/our gem.  Thanks for looking into this.  Sorry for being slow here...